### DISCUSSION TITLE
CMU 15-131 计算机科学家的伟大实用思想 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#26] [Announcements]
# CMU 15-131 计算机科学家的伟大实用思想 | hackway.org

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/cs/freshman/tools/cs15131#comments

**Comment by rubyfrost101** [comment]
2022年春的链接地址是 berkeley 的官网，这个课程是 berkeley 的，而不是 cmu 的？
